src/InputOutput/writers.jl

```diff
@@ -45,11 +45,14 @@ end
 function HDF5Writer(
     filename::AbstractString,
     context::ClimaComms.AbstractCommsContext,
+    overwrite::Bool = true,
```

Comment:
```suggestion
    context::ClimaComms.AbstractCommsContext;
    overwrite::Bool = true,
```
this might make more sense as a keyword argument.